extensions/data-workspace/src/dialogs/newProjectQuickpick.ts

```diff
@@ -52,6 +52,12 @@ export async function createNewProjectWithQuickpick(workspaceService: WorkspaceS
 	if (defaultProjectSaveLoc) {
 		browseProjectLocationOptions.unshift(defaultProjectSaveLoc.fsPath);
 	}
+
+	// if there's an open folder, add it for easier access. If there are multiple folders in the workspace, default to the first one
+	if (vscode.workspace.workspaceFolders) {
+		browseProjectLocationOptions.unshift(vscode.workspace.workspaceFolders[0].uri.fsPath);
```

Comment:
Won't this add the workspace folder at the top, even if it had default project save location? Shouldn't the project save location stay on top?

Reply:
swapped them in the latest commit so that the default location is first, above the open folder